### ISSUE TITLE
[Feature Request] Make showing number before book title, optional

### ISSUE THREAD

**Issue by jonaballen** [gotson/komga#686]
### Is your feature request related to a problem? Please describe.
A lot of series does not have a natural order, as an example: I place singular albums in folders named after the creator/author and it feels just wrong to display the book titles prefixed with a number. It would be nice to have an option to hide the number

Personally, I prefer the title in all ist glory, without adding a sort order digit in front of it. For me, the title should remain unaltered, both for form and aesthetics

### Describe the solution you'd like
Add option to show or hide the book number on a system, library and/or series level.

**Comment by gotson** [comment]
You should not use series for things that are not series. Read lists are made for that.

**Comment by Kussie** [comment]
I personally would like to see something like this as well.  Be it a library toggle or an overall toggle, could still keep the functionality there, just have the option hide the display of it on the frontend.

My use case is kind of unique, but i basically strip out chapter/volume names and just use a number and all my files are named as "<series name> - 01.cbz" etc.  So in Komga they display as "1 - <Series Name> - 01"

**Comment by jonaballen** [comment]
Well komga treats everything as series, it scans a library from a file system and every set of files located in a folder is treated as a series.
It doesnt manage one-off albums, graphic novels etc as singel items, it still has a number, in front.
But basically, it more a convenience thing and an aesthetic one.

**Comment by gotson** [comment]
> It doesnt manage one-off albums, graphic novels etc as singel items, it still has a number, in front.

they need to be placed in their own folder

**Comment by jonaballen** [comment]
> they need to be placed in their own folder
Which I do in the vast majority of the cases, however there is still a number "1." in front of the title

I also have some other folders with miscellaneous books, which I have collected in the same folder, there are other cases, where I have collected books that dont have a natural reading order. And users will have many ways of arranging their collections.
If by adding a few options would allow for some variation to organization, the threshold to use komga as your basic driver would be lowered.
